server/src/test/java/org/elasticsearch/ExceptionSerializationTests.java

```diff
@@ -353,10 +354,12 @@ public void testSearchContextMissingException() throws IOException {
     }
 
     public void testCircuitBreakingException() throws IOException {
-        CircuitBreakingException ex = serialize(new CircuitBreakingException("I hate to say I told you so...", 0, 100));
-        assertEquals("I hate to say I told you so...", ex.getMessage());
+        CircuitBreakingException ex = serialize(new CircuitBreakingException("Too large", 0, 100, CircuitBreaker.Durability.TRANSIENT),
+            randomFrom(Version.V_7_0_0_alpha1));
```

Comment:
I don't think we need the `randomFrom` for a single version here? Especially since this isn't planned for backporting this can be `Version.CURRENT` right?

Reply:
Thanks. I've removed the randomization now.